README.md

```diff
@@ -54,10 +54,19 @@ Stream<Row> rowStream = Parsers
   .parse(FileSource.of(new FileInputStream("myFile.xlsx")));
 ```
 
-With CSV files, it is possible to specify multiple parameters like the charset or the escape character:
+With CSV files, it is possible to specify multiple parameters the escape character:
```

Comment:
`With CSV files, it is possible to specify multiple parameters like the escape character:`